docs/tutorial/quickstart.md

```diff
@@ -12,16 +12,15 @@ Create a new Django project named `tutorial`, then start a new app called `quick
 
     # Create a virtualenv to isolate our package dependencies locally
     virtualenv env
-    source env/bin/activate  # On Windows use `env\Scripts\activate`
+    . env/bin/activate  # On Windows use `env\Scripts\activate`
```

Comment:
I'd prefer `source` in either case. Believe `.` may not work across all terminal envs, and `source` is also clearer.